src/gam.py

```diff
@@ -2244,11 +2296,19 @@ def doPrintGuardians():
         states = [u'COMPLETE', u'PENDING', u'GUARDIAN_INVITATION_STATE_UNSPECIFIED']
       i += 1
     elif sys.argv[i].lower() == u'states':
-      states = sys.argv[i+1].split(u',')
+      states = sys.argv[i+1].lower().replace(u'_', u'').split(u',')
+      for j, state in enumerate(states):
+        if state not in GUARDIAN_STATE_MAP:
+          print u'ERROR: state must be one of %s; got %s' % (u', '.join(GUARDIAN_STATE_MAP), state)
```

Comment:
The discovery doc already handles making sure state is an acceptable value. If we also check and Google adds a new state like "REJECTED" then we need to upgrade in order for GAM to use the new state. Better to let the discovery doc handle the check.